@navikt/core/react/src/layout/grid/HGrid.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { forwardRef, HTMLAttributes } from "react";
+import cl from "clsx";
+import {
+  getResponsiveProps,
+  getResponsiveValue,
+  ResponsiveProp,
+  SpacingScale,
+} from "../utilities/css";
+
+export interface HGridProps extends HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode;
+  /**
+   * Number of columns to display. Can be a number, a string with a unit or tokens for spesific breakpoints.
+   * Sets `grid-template-columns, so 'fr, minmax' etc works`.
```

Comment:
Se over backtics og quotes.

```suggestion
   * Sets `grid-template-columns`, so `fr`, `minmax` etc. works.
```